Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
-gem 'jekyll', '= 2.5.3'
+gem 'jekyll', '~> 3.8'
```

Comment:
Why a `~` version and not `=`?

Reply:
To allow non-major updates that are not expected to introduce breaking changes.

This is in line with our JavaScript dependency management where we use npm's default, i.e. `^` version ranges. If there are good reasons to prefer version pinning that are specific to RubyGems, we can revert to using a pinned version of course.